app/scenes/Search/Search.tsx

```diff
@@ -52,12 +53,17 @@ class Search extends React.Component<Props> {
 
   lastParams: SearchParams;
 
+  lastTitleFilterState: boolean;
```

Comment:
Why store this separately? It should be in params so that the page can be reloaded and keep settings

Reply:
i've fixed it. thanks